tests/test.spec.ts

```diff
@@ -17,12 +17,13 @@ var module = require
 (
   '1'
 )  ;
-import module from ('2');
+import module from '2';
```

Comment:
The fact that parens were allowed made it more difficult to crate a sufficiently restrictive path-matching regex. But having parens around import-paths strings (unless using functional importing, e.g. `import("path")`) is not valid according to spec. I think you can technically do it with CommonJS since imports are synchronous, but it's not allowed with Modules since imports are async.

Long story short: if we don't allow parens around import paths we don't lose any important functionality and restrictive matching is easier.